lib/rubocop/cop/naming/ascii_identifiers.rb

```diff
@@ -4,6 +4,19 @@ module RuboCop
   module Cop
     module Naming
       # This cop checks for non-ascii characters in identifier names.
+      #
+      # @example
+      #     # bad
+      #     def καλημερα
```

Comment:
The examples are not complete ruby code. I think the example should include `end` also.